### ISSUE TITLE
Case-sensitive duplicate package in search results

### ISSUE THREAD

**Issue by allejo** [composer/packagist#1012]
When [searching Packagist for a particular package, `mikey179/vfsstream`](https://packagist.org/search.json?q=vfsstream), I'm getting duplicate results with different case sensitivity.

- `mikey179/vfsstream`
- `mikey179/vfsStream`

The install counts are also different for both of them. Is this intended behavior for backward-compatibility or some other problem?

**Comment by stof** [comment]
this might be a side-effect of the cleanup performed by @Seldaek for existing packages on Packagist (switching them to lowercase in the DB). Maybe the Algolia search index was not properly cleaned and so still contains the previous non-lowercase name.

**Comment by pixelbrackets** [comment]
Side effect: When I try to replace packages with uppercase to lowercase I get the same uppercase package name again. Fixing the deprecation warning is not possible without manual editing the `composer.json` file (bad practice).

Example: Package »natxet/cssmin« → https://packagist.org/?query=natxet%2Fcssmin

* Expected behaviour: Install »natxet/cssmin«
* Actual behaviour: Installs »natxet/CssMin«

``` bash
$: composer remove natxet/CssMin

Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 0 installs, 0 updates, 1 removal
  - Removing natxet/cssmin (v3.0.6)
Generating autoload files
All packages are up to date.
```

``` bash
$: composer require natxet/cssmin

Using version ^3.0 for natxet/CssMin
./composer.json has been created
Deprecation warning: require.natxet/CssMin is invalid, it should not contain uppercase characters. Please use natxet/cssmin instead. Make sure you fix this as Composer 2.0 will error.
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
  - Installing natxet/cssmin (v3.0.6): Downloading (100%)         
Writing lock file
Generating autoload files
All packages are up to date.
```

``` bash
$: cat composer.json 

{
    "require": {
        "natxet/CssMin": "^3.0"
    }
}
```

**Comment by Seldaek** [comment]
Fixed the duplicate search result and the composer require issues. Thanks for reporting.
